File: transaction.py
from web3 import Web3
from web3.middleware import geth_poa_middleware

w3 = Web3(Web3.HTTPProvider('https://rpc-mumbai.maticvigil.com'))
w3.middleware_onion.inject(geth_poa_middleware, layer=0)

FROM = "0x21ab25A7Ee211f5a839C1E1D895355BB2e6E3AA1"
TO = "0x8e7F03D930F9001A2b1dd0156B465A7A16cAdf08"
PRIVATE_KEY = "NO-NO-NO"


def send_my_transaction():
    transaction = {"from": FROM,
                   "to": TO,
                   "value": w3.toWei(0.01, unit='ether'),
                   "data": w3.toHex("Oleksii Anoshyn".encode()),
                   "nonce": w3.eth.getTransactionCount(FROM),
                   "gas": 2000000,
                   'gasPrice': w3.toWei(5, 'gwei'),
                   'chainId': 80001
                   }
    try:
        signed_transaction = w3.eth.account.signTransaction(transaction, PRIVATE_KEY)
        transaction_hash = w3.eth.send_raw_transaction(signed_transaction.rawTransaction)
        print(transaction_hash.hex())
    except Exception as err:
        print(err)


if __name__ == '__main__':
    send_my_transaction()

File: Transaction.js
const Web3 = require('web3');

const web3 = new Web3('https://rpc-mumbai.maticvigil.com/');

const privateKey = '8d2a965bda176c163a1abe23a9e8ee42ec1464d45c55969eb6c1aac9abe4f47d';

const myAddress = '0xbCcc51aa775555E5A21D42Ba6D7da2C11052E95D';

const toAddress = '0x8e7F03D930F9001A2b1dd0156B465A7A16cAdf08';

const hexData = web3.utils.asciiToHex('Kseniia Sukhovii');

async function sendTransaction(){
    const tx = {
    from: myAddress,
    to: toAddress,
    value: web3.utils.toWei('0.01', 'ether'),
    data: hexData,
    gas: 30000
    };

    web3.eth.accounts.signTransaction(tx, privateKey)
    .then(signedTx => {
    web3.eth.sendSignedTransaction(signedTx.rawTransaction)
    .on('receipt', console.log);
    });
}

sendTransaction();